### ISSUE TITLE
Post being duplicated

### ISSUE THREAD

**Issue by kimclow** [a8cteam51/query-loop-load-more#26]
Potentially linked to #23 and #5

> When I click the load more, only three posts load (not sure where that number is set because it’s not anywhere in my reading settings or anywhere else) and the three posts are the most recent posts (duplicates) rather than the next three in line.

Reported here: https://wordpress.org/support/topic/post-being-duplicated/ 

I am asking for more information on the Support page. 

> - Is the issue visible on the frontend by any chance? Could you share URL with us?
> - Could you share what version of WordPress you’re running?
> - What browser and version are you using?
> - Could you share a list of your plugins you have active on the site?
> - Are the three posts that are duplicating set to “sticky” by any chance?

**Comment by kimclow** [comment]
The user has replied: 

We can view the error [here](https://glencanyon.mysites.io/latest/).

Running:
– WP version: wp 6.6.2
– Gutenberg 19.5.0
– Chrome
– Mac OS
- The list of plugins are [here](https://www.dropbox.com/scl/fi/9d1w1pb4rp4uyf550ipu4/Screenshot-2024-10-31-at-2.39.39-PM.png?rlkey=drf0d9bcnkq622sjsot5rvr1j&dl=0).
- The posts are not sticky. Using query block with the default query type.

**Comment by gin0115** [comment]
The user has reverted back to the regular archive template, so it's not possible to see what's happening here.

I fixed the issue of duplicated posts appearing when I hit the same issue on another site. 
https://github.com/a8cteam51/query-loop-load-more/pull/25

There is still a known issue where having 2 query loops on the page causes weird issues where another loops posts are returned and can cause paging issues. I have looked into fixing this, but it requires a serious rethink of how we do this. 

Just tested the current version and it seems to be working still as expected, so unsure what to do here 🤷🏼 
![image](https://github.com/user-attachments/assets/6f718cc7-379b-483e-ac69-5ecb5f4c8fd0)

**Comment by kimclow** [comment]
Hey @gin0115  - thanks so much. Let me go back to the person who reported this and share the update. I'll check they didn't have two loops on the page.

**Comment by kimclow** [comment]
Hey @gin0115 
On #25 is that patch included in the latest version of the site? I'm wondering if he just needs to update it.

**Comment by gin0115** [comment]
Sadly the screenshot of all plugins is cropped so we cant see the version 
![image](https://github.com/user-attachments/assets/848790ad-4d7c-4932-b443-d9e33fd292d0)

BUT if the user had the latest version when reported, it would have that patch in.

**Comment by kimclow** [comment]
Oh sorry what I meant is, is the patch in the latest version of the PLUGIN! Site was a typo. I'll go ahead and ask him to check he's on the latest version to be safe.

**Comment by gin0115** [comment]
yeah, the version currently on both GitHub and .org plugin repos has the patch.

**Comment by gin0115** [comment]
Did we ever hear back?

**Comment by kimclow** [comment]
Hey @gin0115  - no. I'm waiting for them to reply still :(

**Comment by gin0115** [comment]
@kimclowa8c shall we close this out and reopen if the user gets back to us?

**Comment by kimclow** [comment]
Hey @gin0115  - let me message them again and see if we can get some traction.

**Comment by gin0115** [comment]
No problem, keep me posted

**Comment by gin0115** [comment]
Guessing we can close this one out now

![Image](https://github.com/user-attachments/assets/f429d8fa-bad9-4a88-9aec-e71d84032f88)

Feel free to tag me in any others that crop up :)

**Comment by kimclow** [comment]
Thanks @gin0115
